process_jartic_archives_fast.py

```
@@ -0,0 +1,217 @@
+                            sensor,
+                            start_date,
+                            end_date
+                        ):
+                            measurements.append(
+                                f"{measurement.timestamp.isoformat()},{location.id},{location.name},"
+                                f"{location.coordinates.latitude},{location.coordinates.longitude},"
+                                f"{sensor.parameter.value},{measurement.value},{sensor.unit.value}\n"
+                            )
+                            
+                    except Exception:
+                        pass  # Skip sensor errors
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)